packages/migrate/src/commands/DbPull.ts

```diff
@@ -204,8 +206,9 @@ Set composite types introspection depth to 2 levels
       )
       .run()
 
-    // Re-Introspection is not supported on MongoDB
+    const schemasFromCliArgs = args['--schemas']?.split(',')
```

Comment:
Is this used anywhere?

Reply:
You mean `schemasFromCliArgs`? Yes it's passed to the introspection engine below

Reply:
Found it.